modules/cnf-configuring-huge-pages.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//CNF-78
+// * scalability_and_performance/cnf-configuring-huge-pages.adoc
+
+[id="cnf-configuring-huge-pages_{context}"]
+= Configuring huge pages
+
+Nodes must pre-allocate huge pages used in an {product-title} cluster. Use the
+Performance Addon Operator to allocate huge pages on a specific node.
+
+{product-title} provides a method for creating and allocating huge pages.
+Performance Addon Operator provides an easier method for doing this using the PerformanceProfile, as described in
+xref:../modules/configuring-huge-pages.adoc[Configuring huge pages].
+
+For example, in the `hugepages` `pages` section of the PerformanceProfile,
+you can specify multiple blocks of `size`, `count`, and, optionally, `node`, as shown here:
```

Comment:
I would drop "as shown here" and just use a : as a lead in

Reply:
Change made.